packages/node-app/src/controllers/giveaway.controller.ts

```diff
@@ -305,6 +329,19 @@ export const updateParticipant = async (req: Request, res: Response) => {
     }
     await giveaway.save();
 
+    const { id, title, description, prize, rules, image } = giveaway;
+    await GiveawayLog.create({
+      action: 'update-giveaway-participant',
+      author: req.user.email,
+      giveaway_id: id,
+      changes: {
+        title,
+        description,
+        notified: participant.notified,
+        participant,
+      },
```

Comment:
We can also include rules here just for consistency. Also, I don't think we need the notified directly, it will be contained in the participant obj